src/ui/TextField/TextField.tsx

```diff
@@ -35,13 +35,14 @@ export const TextField = styled('input', {
         fontSize: '$small',
       },
       md: {
-        width: '250px',
-        height: '48px',
+        // width: '250px',
+        // height: '48px',
```

Comment:
@wingfeatherwave ?

Reply:
is this intentinoal or can be removed?

Reply:
Yes, I will remove.  And will resolve the dework button issue.  ty!